deselected_tests.yaml

```diff
@@ -257,7 +257,12 @@ deselected_tests:
   - preprocessing/tests/test_discretization.py::test_nonuniform_strategies[kmeans-expected_2bins1-expected_3bins1-expected_5bins1] >=0.24
 
   # OOB scores in scikit-learn and oneDAL are different because of different random number generators
-  - ensemble/tests/test_forest.py::test_forest_classifier_oob[X1-y1-0.65-array-ExtraTreesClassifier]
+  - ensemble/tests/test_forest.py::test_forest_classifier_oob[X1-y1-0.65-array-ExtraTreesClassifier]  
```

Comment:
Is it possible adopt this test suits for our sklearnex testing add them to `sklearnex/ensemble/tests/test_forest.py?` I mean we can validate OOB with our own tests.

Reply:
Thats a good idea, and definitely something we could do for the next release.

Reply:
let's plan this internally with some  jiras